### DISCUSSION TITLE
zh/guide/advanced/2fa

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#22] [Announcements]
# zh/guide/advanced/2fa

AList v3的文档

https://alist.nn.ci/zh/guide/advanced/2fa.html

**Comment by hubumulity** [comment]
请问手机丢了如何去除两步验证

**Reply by Zynismus16** [reply]
同求

**Reply by xhofe** [reply]
```bash
./alist cancel2fa
```

**Reply by babavoss** [reply]
我输入./alist cancel2fa，显示-ash: ./alist: not found

**Reply by babavoss** [reply]
前面要不要加入其他命令，比如docker exec啥的？

**Reply by sfdzkj** [reply]
要找到alist位置。
我是在群辉第三方套件里 安装的。
cd到   /volume1/@appstore/alist3/bin/ 目录然后再执行  ./alist cancel2fa   重启alist  就可以了。
不过还是建议 后台能关闭。

**Reply by sfdzkj** [reply]
./alist3 cancel2fa

**Comment by amiaoapp** [comment]
还有一个问题，我想绑定到多个设备上。现在只要绑定一个设备就不显示两步验证了

**Comment by BuckMelin** [comment]
有个问题是，我不想用了，那我该如何关闭两部验证？设置里没有提供选项

**Reply by BuckMelin** [reply]
使用./alist cancel2fa没有效果

**Comment by mile000007** [comment]
replit如何关闭输了都没有效果

**Comment by 99477204** [comment]
reading config file: data/config.json
INFO[2023-04-07 11:36:20] load config from env with prefix: ALIST_
INFO[2023-04-07 11:36:20] init logrus...

**Comment by SoulRui** [comment]
我没有听懂如何打开文件夹执行，问一下openwrt怎么清除2FA

**Comment by tzzzer** [comment]
开启两步验证之后，使用fileball的webdav（https）连接的时候并不需要两步验证的验证码是什么原因呢

**Reply by tzzzer** [reply]
也就是otp的验证码

**Reply by xhofe** [reply]
https://www.google.com/search?q=does+webdav+support+2fa

**Comment by sharyuke** [comment]
如果有位置文件，用配置文件取消
```bash
alist --data /etc/alist/ cancel2fa
```
但是我发现，2fa有问题，经常提示验证码错误，我看了时间，有配置时间同步。而且没有针对不同用户配置不同的2fa，配置了之后，所有用户都显示有2fa。
